pkg/golinters/goimports.go

```diff
@@ -27,7 +27,8 @@ func NewGoimports(settings *config.GoImportsSettings) *goanalysis.Linter {
 
 	return goanalysis.NewLinter(
 		goimportsName,
-		"In addition to fixing imports, goimports also formats your code in the same style as gofmt.",
+		"Check import statements are formatted according to the 'goimport' command. "+
+			"Reformat code when the '--fix' option is specified.",
```

Comment:
Did you see there's a column at https://golangci-lint.run/usage/linters/ that tells if the linter has autofix or not (that it will format your code with `--fix` which is the same)? I'm not sure it's needed twice? Also worth mentioning the flag is not the only way apply fixes.